tests/test_optimize.py

```diff
@@ -253,17 +248,6 @@ def test_optimize_partitions_raises_exception_with_cutoff_time() -> None:
     tracker.update_all_partitions([dummy_partition])
     scheduler = OptimizeScheduler(2)
 
-    with pytest.raises(JobTimeoutException):
-        optimize_partitions(
```

Comment:
Why did you remove the call to `optimize_partitions` entirely? Shouldn't the test make the call with a different exception?

Reply:
There is more context in the description of the PR. In short, optimize_partitions does not raise exceptions anymore. So I removed it from this test since this tests purpose is for raised exceptions. The logic for catching exceptions is below that code https://github.com/getsentry/snuba/pull/2955/files#diff-2e9c0212505684117bae2d1a5003a9cae77e63141310ea78ba8fc860d38bc2dfR254-R263